io.catenax.single_level_bom_as_built/RELEASE_NOTES.md

```diff
@@ -3,6 +3,18 @@ All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
 
+## [2.0.0]
```

Comment:
Please don't forget to add date of MS3 here.

Reply:
Will do once we know it!

Reply:
@jonbckr  please add the MS3 date